### ISSUE TITLE
qbmove controller position and stiffness 

### ISSUE THREAD

**Issue by Mohammed20PHD** [NMMI/qbmove-simulink#2]
Dear Colleagues
Kindly, do you have an idea if  I can use a controller Simulink to control qbmove like PID controller
or any controller suggest 
and If I want to control the position and stiffness together is that possible with Simulink?

Best Regards

**Comment by mattiapoggiani** [comment]
Yes both are possible. Abput the control of position and stiffness together, you can use the qbmove block with Equilibrium and Stiffness setting. The first reference will be the position reference while the second will be the stiffness from 0 to 1.
Best

**Comment by Mohammed20PHD** [comment]
Can you help me with more details an example ??
What do you mean exactly?
Thank you very much

**Comment by Mohammed20PHD** [comment]
any update?

**Comment by mattiapoggiani** [comment]
Hi, there is already an example for this kind of control. Please check the qbmove_example.slx Simulink file first. Feel free to write me if there is something you don't understand.
Best

**Comment by Mohammed20PHD** [comment]
Hi, I want to ask, Is it possible to design PID controller with Simulink for Torque Control and Position control. 
And how can I  apply the trajectory tracking for a position?
Best Regards

**Comment by mattiapoggiani** [comment]
Hi, you can do both through Simulink. I don't know what is your confidence with this tool, but, since the qbmove block receives input at every simulation cycle, you can implement whatever external control loop that generate a proper reference to be given as input to the QB block.
The same thing for the trajectory tracking.
